scripts/utils/build.js

```diff
@@ -69,11 +93,12 @@ export async function rollupScript (scriptPath, name, supportsMozProxies = false
                 preventAssignment: true,
                 values: {
                     // @ts-expect-error https://app.asana.com/0/1201614831475344/1203979574128023/f
-                    mozProxies
+                    mozProxies,
+                    'import.meta.injectName': JSON.stringify(platform)
                 }
             }),
             prefixPlugin(prefixMessage)
-        ]
+        ].filter(Boolean)
```

Comment:
This adds complexity when it's not needed.
I'd handle the lack of platform in stubFeatures and just do a noop essentially.